userspace/libscap/engine/savefile/converter/converter.cpp

```diff
@@ -0,0 +1,407 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <driver/ppm_events_public.h>
+#include <converter/types.h>
+#include <converter/results.h>
+#include <converter/debug_macro.h>
+#include <stdarg.h>
+#include <cstdio>
+#include <cassert>
+#include <unordered_map>
+#include <string>
+#include <stdexcept>
+#include <memory>
+
+extern std::unordered_map<conversion_key, conversion_info> g_conversion_table;
```

Comment:
I'd create a separate header for the declaration, ie: `table.h` (private to the converter lib of course), that includes `unordered_map` and declares
```
extern std::unordered_map<conversion_key, conversion_info> g_conversion_table;
```
. 
Then we could include it here and in table.cpp, removing the `unordered_map` include.